pinot-controller/src/main/java/org/apache/pinot/controller/api/resources/PinotQueryResource.java

```diff
@@ -398,18 +414,13 @@ private String getQueryURL(String protocol, String hostName, int port) {
     return String.format("%s://%s:%d/query/sql", protocol, hostName, port);
   }
 
-  public String sendPostRaw(String urlStr, String requestStr, Map<String, String> headers) {
+  public void sendPostRaw(String urlStr, String requestStr, Map<String, String> headers, OutputStream outputStream) {
     HttpURLConnection conn = null;
     try {
-      /*if (LOG.isInfoEnabled()){
+      if (LOGGER.isInfoEnabled()) {
         LOGGER.info("Sending a post request to the server - " + urlStr);
```

Comment:
This should be broker instead of server right?

Reply:
I just uncommented this log. I decided to remove it in https://github.com/apache/pinot/pull/14994